domains/abdallahbari.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Abdallah01win",
+        "email": "abdellah01win@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://portfolio-bf2.pages.dev/"
```

CNAME can't contain protocol or `/`
```suggestion
        "CNAME": "portfolio-bf2.pages.dev"
```